modular_splurt/code/modules/mob/living/say.dm

```diff
@@ -0,0 +1,3 @@
+if (HAS_TRAIT(src, TRAIT_TONGUELESS_SPEECH)) //this exists solely because deprivation helms
+	message = detongueify(message)
+
```

Comment:
```suggestion
/mob/living/treat_message(message)
	if (HAS_TRAIT(src, TRAIT_TONGUELESS_SPEECH)) //this exists solely because deprivation helms
		message = detongueify(message)
	. = ..()
```